### ISSUE TITLE
About multi-gpu training

### ISSUE THREAD

**Issue by XpracticeYSKM** [zhaohengz/LLaMP#3]
Thanks for this good work!

I wonder if I only have a A800(80g), how can I change the hyper-parameters to obtain the same results in your paper？Should I double the learning rate or batch size?

**Comment by zhaohengz** [comment]
Hello,

Thanks for your interest.

Practically, I would recommend you first increase the batch size to maximize the memory consumption, and then increase the lr linearly. For example, if you 2x the batch size, try 2x the lr as well. But zero-shot learning is particularly unstable and you may still need further tuning. The linear-projected lr will be a good start point, though.

**Comment by XpracticeYSKM** [comment]
Thanks for your reponse, I will close this issue.
